strawberry/exceptions.py

```diff
@@ -246,3 +246,14 @@ def __init__(self):
             "that inherits from BaseContext or a dictionary"
         )
         super().__init__(message)
+
+
+class DuplicatedTypeName(Exception):
```

Comment:
@strawberry-graphql/core This is not specific to this PR, but I'm wondering whether we eventually want all our errors to extend a common base exception, so that strawberry errors are easier identifiable and catch-able? (or is this even considered an anti-pattern, since users would be tempted to catch the base strawberry exception instead of more specific ones?)

Reply:
yes, this is the plan, I think we can leave this PR as it is, and I'll update #2027 when this one is merged :)

Reply:
Ah, I see. Thanks Patrick!